HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgammaUnseededSequence_cfi.py

```diff
@@ -18,13 +18,137 @@
+    linkingPSet = cms.PSet(
+        type=cms.string("SuperClusteringMustache"),
+        algo_verbosity=cms.int32(0)
+    ),
+    tracksters_collections = [cms.InputTag("hltTiclTrackstersCLUE3DHigh")], # to be changed to ticlTrackstersCLUE3DEM once separate CLUE3D iterations are introduced
+)
+
+from RecoHGCal.TICL.ticlEGammaSuperClusterProducer_cfi import ticlEGammaSuperClusterProducer
+hltTiclEGammaSuperClusterProducerUnseeded = ticlEGammaSuperClusterProducer.clone()
+
+from Configuration.ProcessModifiers.ticl_superclustering_dnn_cff import ticl_superclustering_dnn
+ticl_superclustering_dnn.toReplaceWith(_SuperclusteringUnseededSequence, 
+                                       cms.Sequence(
+                                            hltTiclTracksterLinksSuperclusteringDNNUnseeded
+                                            + hltTiclEGammaSuperClusterProducerUnseeded
+                                        )
+)
+ticl_superclustering_dnn.toModify(hltTiclEGammaSuperClusterProducerUnseeded,  
+                                  ticlSuperClusters=cms.InputTag("hltTiclTracksterLinksSuperclusteringDNNUnseeded"),
+                                  ticlTrackstersEM=cms.InputTag("hltTiclTrackstersCLUE3DHigh"),
+                                  layerClusters=cms.InputTag("hltHgcalMergeLayerClusters"))
+
+# Ticl mustache
+from Configuration.ProcessModifiers.ticl_superclustering_mustache_ticl_cff import ticl_superclustering_mustache_ticl
+ticl_superclustering_mustache_ticl.toReplaceWith(_SuperclusteringUnseededSequence, 
+                                       cms.Sequence(
+                                            hltTiclTracksterLinksSuperclusteringMustacheUnseeded
+                                            + hltTiclEGammaSuperClusterProducerUnseeded
+                                        )
+)
+ticl_superclustering_mustache_ticl.toModify(hltTiclEGammaSuperClusterProducerUnseeded, 
+                                            ticlSuperClusters=cms.InputTag("hltTiclTracksterLinksSuperclusteringMustacheUnseeded"),
+                                            ticlTrackstersEM=cms.InputTag("hltTiclTrackstersCLUE3DHigh"),
+                                            layerClusters=cms.InputTag("hltHgcalMergeLayerClusters"))
+
+# Alpaka & DNN
+(alpaka & ticl_superclustering_dnn).toReplaceWith(_HgcalLocalRecoUnseededSequence, 
+                    cms.Sequence(
+                        hltHgcalDigis
+                        + hltHGCalUncalibRecHit
+                        + hltHGCalRecHit+hltParticleFlowRecHitHGC
+                        + hltHgcalSoARecHitsProducer
+                        + hltHgcalSoARecHitsLayerClustersProducer
+                        + hltHgcalSoALayerClustersProducer
+                        + hltHgCalLayerClustersFromSoAProducer
+                        + hltHgcalLayerClustersHSci
+                        + hltHgcalLayerClustersHSi
+                        + hltHgcalMergeLayerClusters
+                        ) 
+)
```

Comment:
How does this differ from 
https://github.com/cms-sw/cmssw/blob/a85bda57ecb39ceb07427c917c4d4dc63594c9cb/HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgammaUnseededSequence_cfi.py#L30-L45?

Don't the aforementioned lines cover the heterogeneous workflow more in general? Maybe I am missing how the `ticl_superclustering_dnn` modifier changes anything here.

Reply:
No, I think you're right. In fact all the `(alpaka & ticl_superclustering)` modifiers are pointless, as `alpaka` and `supercluster` modify independent sequences. Thanks